pkgs/top-level/all-packages.nix

```diff
@@ -29717,4 +29717,6 @@ in
   lc3tools = callPackage ../development/tools/lc3tools {};
 
   zktree = callPackage ../applications/misc/zktree {};
+
+  zydis = callPackage ../development/libraries/zydis { };
```

Comment:
Please sort this into a fitting category and sort there alphabetically.

Reply:
Oops, I just realized this file is grouped by categories. I made the requested change, thanks.